.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
I think would be great to add the rules from other projects in example antifraud

This will help us, a little bit to stadardize what we are reviewing

For example in other projects we have use root statements as in import section, instead of inline for every time we use them

wdyt